include/wlr/types/wlr_seat.h

```diff
@@ -252,10 +252,25 @@ struct wlr_seat {
 		struct wl_signal request_set_cursor;
 
 		// wlr_seat_request_set_selection_event
+
+		// Called when an application _wants_ to set the selection (user copies some data).
+		// More specifically, the application wants to set a wlr_data_source object for the selection.
+		// This is what a compositor usually listens for.
 		struct wl_signal request_set_selection;
+		// Called after the requested data source is set for the selection
+		// i.e. after you call wlr_seat_set_selection.
+		// When this is called, the data is already copied. It's more of a confirmation.
```

Comment:
"the data is already copied" → no, data hasn't been copied when selection is set.

"It's more of a confirmation." → this is a little bit vague